cleanup/lib/force-failover.js

```diff
@@ -0,0 +1,92 @@
+let file;
+let target;
+let leaderCid;
+let shardIndex;
+
+(function() {
+  if (3 < ARGUMENTS.length) {
+    file = ARGUMENTS[0];
+    target = ARGUMENTS[1];
+    leaderCid = ARGUMENTS[2];
+    shardIndex = ARGUMENTS[3];
+
+  } else {
+    print("usage: force-failover.sh --server.endpoint LEADER-AGENT INPUT-FILE TARGET-SERVER LEADER-CID SHARD-INDEX");
+    return;
+  }
+
+  try {
+    let role = require("@arangodb").db._version(true).details.role;
+
+    if (role === "AGENT") {
+      let agency = arango.POST('/_api/agency/read', [
+        ["/"]
```

Comment:
Do we really want to read the **WHOLE** agency to test if this is the leader? :D you could also use '/_api/agency/config' and compare `leaderId` with `configuration.id`.

Reply:
oh this is a good point !
I think it is enough that this agent actually responds, so we could read `/foo` instead which is comparably cheap.
However this is not good in all scripts, so I would like to extract this into a different PR